src/main/java/com/simibubi/create/content/schematics/cannon/SchematicannonScreen.java

```diff
@@ -46,7 +46,8 @@ public class SchematicannonScreen extends AbstractSimiContainerScreen<Schematica
 
 	private final String _slotGunpowder = "gui.schematicannon.slot.gunpowder";
 	private final String _slotListPrinter = "gui.schematicannon.slot.listPrinter";
-	private final String _slotSchematic = "gui.schematicannon.slot.schematic";
+	private final String _slotSchematicEnabled = "gui.schematicannon.slot.schematic";
```

Comment:
Should I migrate  it to "gui.schematicannon.slot.schematicEnabled"? is enough just changing the id?